File: official/docs/java/webhooks/CreateWebhook.java
import com.easypost.EasyPost;
import java.util.*;

public class CreateWebhook {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Map<String, Object> paramMap = new HashMap<String, Object>();
        paramMap.put("url", "example.com");

        Webhook webhook = Webhook.create(paramMap);

        System.out.println(webhook);
    }
}

File: official/docs/csharp/webhooks/CreateWebhook.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task CreateWebhook()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        Webhook webhook = await Webhook.Create(
            new Dictionary<string, object>() {
                { "url", "example.com" }
            }
        );

        new TestOutputHelper().WriteLine(JsonConvert.SerializeObject(webhook, Formatting.Indented));
    }
}
